react/components/InfoCard/index.js

```diff
@@ -168,6 +176,36 @@ const InfoCard = ({
 
   const subheadClasses = `${handles.infoCardSubhead} t-body mt6 c-on-base ${alignToken} mw-100`
 
+  const infoCardRef = useRef(null)
+
+  const { push } = usePixel()
+
+  useOnView({
+    ref: infoCardRef,
+
+    onView: () => {
+      const promotionEventData =
+      analyticsProperties === 'provide'
+        ? {
+            id: promotionId,
+            name: promotionName,
+            position: promotionPosition,
+          }
+        : undefined
+
+      if (analyticsProperties === 'none') {
+        return
+      } else {
+        push({
+          event: 'promoView',
+          promotions: [promotionEventData],
+        })
+      }
+    },
```

Comment:
analyticsProperties can have which values? If it only has 'provide' and 'none,' you can improve your onView function.

```suggestion
    onView: () => {
      if (analyticsProperties === 'none') {
        return
      }

      const promotionEventData = {
        id: promotionId,
        name: promotionName,
        position: promotionPosition,
      }
        
        push({
          event: 'promoView',
          promotions: [promotionEventData],
        })
    },
```
Does this suggestion make sense for you?